packages/docs/source/components/tab.md

```diff
@@ -0,0 +1,306 @@
+      <p>Tab Panel 0: Agents</p>
+    </div>
+    <div id="tab-1-1" tabindex="0" role="tabpanel" aria-labelledby="tablistitem-1-1">
+      <p>Tab Panel 1: People</p>
+    </div>
+    <div id="tab-1-2" tabindex="0" role="tabpanel" aria-labelledby="tablistitem-1-2" hidden="">
+      <p>Tab Panel 2: Settings</p>
+    </div>
+    <div id="tab-1-3" tabindex="0" role="tabpanel" aria-labelledby="tablistitem-1-3" hidden="">
+      <p>Tab Panel 3: Import</p>
+    </div>
+  </div>
+</div>
+```
+
+This can be accomplished by changing the following custom properties:
+
+<figure class="ods-table--figure">
+  <table class="ods-table">
+    <caption>Odyssey takes care to ensure apropriate keyboard navigation for the tab component.</caption>
+    <thead>
+      <tr>
+        <th scope="column">CSS variable</th>
+        <th scope="column">Description</th>
+      </tr>
+    </thead>
+    <tbody>
+      <tr>
+        <td><code>--ods-indicator-width</code></td>
+        <td>The width of the tab indicator</td>
+      </tr>
+      <tr>
+        <td><code>--ods-indicator-pos-x</code></td>
+        <td>The position of the tab indicator along the x-axis</td>
+      </tr>
+    </tbody>
+  </table>
+</figure>
+
+## Accessibility
+
+### Keyboard support
+<figure class="ods-table--figure">
+  <table class="ods-table">
+    <caption>Odyssey takes care to ensure apropriate keyboard navigation for the tab component.</caption>
+    <thead>
+      <tr>
+        <th scope="column">Key</th>
+        <th scope="column">Behavior</th>
+      </tr>
+    </thead>
+    <tbody>
+      <tr>
+        <th scope="row" rowspan="2"><kbd>Tab</kbd></th>
+        <td>When focus moves in to <code>tablist</code> the focus is placed on the first <code>tab</code> element.</td>
+      </tr>
+      <tr>
+        <td>When focus is in to <code>tablist</code> focus moves in to the active <code>tabpanel</code>.</td>
+      </tr>
+      <tr>
+        <td><kbd>Enter</kbd> <kbd>Space</kbd></td>
+        <td>When focus is placed on a tab, the corresponding <code>tabpanel</code> is activated/displayed.</td>
+      </tr>
+      <tr>
+        <th scope="row" rowspan="2"><kbd>Right Arrow</kbd></th>
+        <td>When focus is within the <code>tablist</code> the next tab is selected.</td>
+      </tr>
+      <tr>
+        <td>If the last tab is focused the focus is moved to the first tab.</td>
+      </tr>
+      <tr>
+        <th scope="row" rowspan="2"><kbd>Left Arrow</kbd></th>
+        <td>When focus is within the <code>tablist</code> the previous tab is selected.</td>
+      </tr>
+      <tr>
+        <td>If the first tab is focused the focus is moved to the last tab.</td>
+      </tr>
+      <tr>
+        <td><kbd>Home</kbd></td>
+        <td>If a tab has focus, the focus is moved to the first tab.</td>
+      </tr>
+      <tr>
+        <td><kbd>End</kbd></td>
+        <td>If a tab has focus, the focus is moved to the last tab.</td>
+      </tr>
+    </tbody>
+  </table>
+</figure>
+
+
+## Content Guidelines
```

Comment:
I think we should move these way up the page, and maybe re-title to Usage?

Reply:
Done